### DISCUSSION TITLE
Can't install Skyline

### DISCUSSION THREAD

**Discussion by GSZA137** [skyline-emu/skyline#840] [Q&A]
I am able to Download everything just fine, and when i go to install, it says to update and it gives an error when i try (conflict with an older version), but i don't have any app downloaded and i can't find anything related to (Skyline-emu-app.release) to delete and be able to install, is there another name to find or any other way to fix this?

**Comment by mateoalfaro** [comment]


**Reply by GSZA137** [reply]
I can't even install, even if i delete and download again

**Comment by GSZA137** [comment]
Really hoping for help and be able to get this!

**Comment by DavForeves** [comment]
I had this issue but you have to delete the skyline on your phone currently
in order to download the new one because since there are new code the phone
senses the same app but with different features that's why that happens

**Comment by Bosh000** [comment]
Make sure you have uninstalled the old version of the app. If you still can't install the new version then restart your device. The app should install after restarting your phone. (This worked for myself)

**Reply by GSZA137** [reply]
Didn't work sadly

**Comment by supposede** [comment]


**Reply by GSZA137** [reply]
I do know that it is in conflict with an older version of the app, the problem is that i can't find that supposed version anywhere even with a file manager so i can't delete.

**Reply by Bosh000** [reply]
Try restarting your device and then installing the apk

**Reply by GSZA137** [reply]
I tried and the same thing happened.

**Comment by GSZA137** [comment]
Seems like some didn't really understand what i meant, i will explain properly: i do know that i have to delete any other version of the app, but the thing is, i can't find ANYTHING related to Skyline, i don't have any old version installed, and i can't find any file related to this, that's it.

**Reply by Nacolla** [reply]
Okay, what is the Android version of your phone?

**Reply by GSZA137** [reply]
12

**Comment by GSZA137** [comment]
I think it is probably something related to an -already deleted- old verison that is left on the phone, but i can't find it searching for names like skyline or app release, etc, is there any other name to go by to find this probably saved data?

**Reply by ghost** [reply]
if u have root, go to root data>data and scroll down until u should see skyline folder, and delete that, but usually it gets deleted when uninstalling the app

**Reply by GSZA137** [reply]
Sorry for the late response, but i don't have root, is it the only way if i can't find normally?
